packages/@webex/plugin-meetings/src/breakouts/README.md

```diff
@@ -119,8 +119,11 @@ breakout.remove(participant)
 // This can only be done when breakouts are inactive
 breakout.delete()
 
-// Create a breakout session
-breakouts.create(sessionName)
+// Create breakout sessions
+breakouts.create(sessions)
+
+// Delete all breakout sessions
+breakouts.clearSessions()
```

Comment:
arent we calling them as rooms / sessions / meetings 

sessions looks little odd here

Reply:
https://sqbu-github.cisco.com/pages/WebExSquared/breakout/api/resource/createBreakoutSessions.html

I checked the design doc,  call the breakout as sessions.